apps/api/src/app/events/dtos/trigger-event-to-all-request.dto.ts

```diff
@@ -41,4 +43,26 @@ export class TriggerEventToAllRequestDto {
   @IsString()
   @IsOptional()
   transactionId?: string;
+
+  @ApiProperty({
+    description: `It is used to display the Avatar of the provided actor's subscriber id or actor object.
+    If a new actor object is provided, we will create a new subsciber in our system
+    `,
+    oneOf: [
+      {
+        $ref: getSchemaPath(SubscriberPayloadDto),
+      },
+      {
+        type: '[SubscriberPayloadDto]',
+        description: 'List of actor objects',
+      },
+      { type: 'string', description: 'Unique identifier of a subscriber in your systems' },
+      {
+        type: '[string]',
+        description: "List of actor's subscriber identifiers",
+      },
```

Comment:
the same about types here, please refer to `apps/api/src/app/events/dtos/trigger-event-request.dto.ts`